src/modules/list.marko

```diff
@@ -230,130 +297,449 @@
+                            >
+                                <use href="#icon-money-back-guarantee-16"/>
+                            </svg>
+                        </span>
+                    </div>
+                </li>
+            </ul>
+        </div>
+    </highlight-code>
+    <h3 id="list-default">
+        List with fieldset
+    </h3>
+    <p>
+        For an interactive list, you can add a fieldset with radios or checkboxes
+    </p>
+    <div class="demo">
+        <div class="demo__inner">
             <div class="list">
-                <ul>
-                    <li>
-                        <div>
-                            <span class="list__leading">
-                                <svg aria-hidden="true" class="icon icon--16" height="16" width="16">
-                                    <icon-symbol name="file-16"/>
+                <fieldset>
+                    <legend>Choose an Option</legend>
+                    <div>
+                        <span class="radio list__leading">
+                            <input
+                                class="radio__control"
+                                id="list-radio-1"
+                                type="radio"
+                                value="1"
+                                name="list-radio-group"
+                            >
+                            <span class="radio__icon" hidden>
+                                <svg
+                                    class="radio__unchecked"
+                                    height="18"
+                                    width="18"
+                                    aria-hidden="true"
+                                >
+                                    <use href="#icon-radio-unchecked-18"/>
                                 </svg>
-                            </span>
-                            <span class="list__body">
-                                Item 1
-                            </span>
-                        </div>
-                    </li>
-                    <li>
-                        <div>
-                            <span class="list__leading">
-                                <svg aria-hidden="true" class="icon icon--16" height="16" width="16">
-                                    <icon-symbol name="file-16"/>
+                                <svg
+                                    class="radio__checked"
+                                    height="18"
+                                    width="18"
+                                    aria-hidden="true"
+                                >
+                                    <use href="#icon-radio-checked-18"/>
                                 </svg>
                             </span>
-                            <span class="list__body">
-                                Item 2
+                        </span>
+                        <label class="list__body" for="list-radio-1">
```

Comment:
I never really liked this nesting. `list__body` feels like a descendent of `list` even though it's specific to `item`. It feels like `list__item-body` may be better, but that ship may have sailed?

Reply:
I had no issue doing `list__item-body` when I put up the original PR.
I had asked that if we should go `list__item-leading` in another thread and the conclusion was to go `list__leading` without the item. I believe the same applies for body since they're both siblings.